src/test/java/Search.java

```diff
@@ -0,0 +1,105 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    String songName = "Dark Days";
+    //use all cases- exact spelling of song name, ingnore trailing and hading white space
+    String [] list = {"Dark Days"," Dark Days ","Dark Days "};
+
+    String [] caseSenList = {"darks day", "darKs DAy"};
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() throws InterruptedException{
+
+//        login();
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < list.length; i++) {
+            //System.out.println(list[i]);
+            searching.searchBox(list[i]);
+        }
+        //check for the results
+        resultFromSearch();
+    }
+    @Test(enabled = true,priority=1)
+    public void isSongSearchwithSentiveCase() {
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        for (int i=0; i < caseSenList.length; i++) {
+            searching.searchBox(caseSenList[i]);
+        }
+        songResultpage();
+        //negativeSearchResult();
+        //Assert.assertFalse(negativeSearchResult(),"should not shown");
+        Assert.assertEquals(negativeSearchResult(), false);
+
+    }
+     //expect to see result
+     public void resultFromSearch(){
+         //*[@id='searchExcerptsWrapper']//span[text()="Dark Days"]
+         WebElement songresult=driver.findElement(By.xpath("//*[@data-testid='song-excerpts']//button[@data-test='view-all-songs-btn']"));
+         songresult.click();
+
+         WebElement isThatCorrectSong = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='title']"));
+         Assert.assertEquals(isThatCorrectSong.getText(), "Dark Days");
+
+         WebElement isThatCorrectartist = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='artist']"));
+         Assert.assertEquals(isThatCorrectartist.getText(), "Grav");
+
+         WebElement isThatCorrectAlbum = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//td[@class='album']"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/209536590-277f20b5-3298-4e48-b11a-9ac110943407.png)

Reply:
I moved that to SearchPage (POM)